.github/CONTRIBUTING.md

```diff
@@ -15,12 +15,45 @@ To contribute,
 
 ## Setting up the Local Development Environment
 
+### Get the source code
 1. [Fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) this repository
 2. Open your favorite command line tool and navigate to the directory you wish to clone this repository to: `cd /path/to/clone`
 3. [Clone](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) your fork: `git clone git@github.com:{your username}/hospitalrun-frontend.git`
 4. Navigate to the hosptialrun-frontend directory: `cd hospitalrun-frontend`
-5. Install dependencies: `yarn`
-6. Run the application `yarn start`
+
+### Configure CouchDB
+CouchDB is the server side database which data from the frontend will sync to. In order to login
+to HospitalRun, CouchDB is required. For convienence, we have added a docker compose file in the
+root of this project to help launch CouchDB. However, you could install and run CouchDB in any way you wish.
+
+The following directions will be for running CouchDB via Docker Compose.
+
+1. Install [Docker](https://docs.docker.com/get-docker/)
+2. Install [Docker Compose](https://docs.docker.com/compose/install/)
+3. Run `docker-compose up` in the root directory.
+> This should launch a new CouchDB instance on `http://localhost:5984` and create a default admin
+> user with a username of `admin` and password of 'password'
+4. Launch `http://localhost:5984/_utils` to view Fauxton.
```

Comment:
I think it'd be friendly to suggest, "Open in a new incognito window to avoid getting logged out during your development."